README.md

```diff
@@ -1,2 +1,92 @@
-# gpii-devpmt
-Development Preferences Test Tool and other Management Tools
+# Development Preferences Test Tool and other Management Tools
+
+This repository contains a web based tool for editing preferences safes and
+sets as part of the larger eco-system of GPII modules. It contains resuable
+routines and widgets as well that other preference editing user interfaces
+can be based on.  While it's changed during the course of development and
+testing, it's fundamental design is still based on these
+[original designs](https://drive.google.com/open?id=0Bxy2B0Y99qCubGVUSlNCRFU3d0U).
+
+## Requirements
+
+The base requirements for building this project are the same as other core
+GPII projects and the setup is detailed in
+[Setting Up Your Development Environment](http://wiki.gpii.net/w/Setting_Up_Your_Development_Environment).
+
+Additionally this project currently relies on a running preferences server using a
+branch containing more API's for the PPT. [GPII-2966](https://github.com/GPII/universal/pull/634).
+
+## Installation
+
+Building the PPT follows the typical set of `git` and `npm` commands.
+Running the PPT depends on having some other external services running. These
+include an instance of CouchDB, a GPII Preferences server, and optionally a
```

Comment:
It would be nice to link to docs of how to set up GPII Preferences server and Redis server if there's one.

Reply:
I've added links and some docker quickstarts for these.